### PR TITLE
I think task 3.5 is done

### PR DESCRIPTION
**Pull request by Masoud-Rad**

### CODE REVIEW

File: models/api.models.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+exports.returnEndpoints=()=>{
+
+    
+     return fs.readFile("../endpoints.jason",'utf8').then((res)=>{
+        const data= JSON.parse(res);
+        return data;
+    }).catch(()=>{})
+}
```
Comment (johnhutcheon) [inline models/api.models.js:13]: We don't actually need a model here for this ticket, everything can be done in the controller 🙂

Comment (Masoud-Rad) [reply models/api.models.js:13]: CHEERS, I WILL EDITE IT

File: app.js [hunk 2]
```diff
@@ -11,7 +11,9 @@ app.get('/api', getDescription);
 app.get("/api/topics",getTopics);
 app.get("/api/articles",getArticles);
 
-
+app.all("*", (req,res)=>{
+    res.status(404).send({ msg: "NOT FOUND!" })
+})
 app.use((error, req, res, next) => {
     res.status(error.status).send({ msg: error.msg })
 })
```
Comment (johnhutcheon) [inline app.js:19]: We'd need an endpoint in here that links to getDescription in the controller 👍

**Review by johnhutcheon — COMMENTED** [review]
Hey Mas! Please see my comments. We don't need a model for this one as everything can be done in the controller. You'll also just need to add an endpoint in your app (something like /api/getApi would be good!). And make sure you add a test to ensure it's returning the information 🙂

**Pull request merged**